src/qibo/quantum_info/metrics.py

```diff
@@ -667,8 +666,8 @@ def expressibility(
     order: Optional[Union[int, float, str]] = 2,
     backend=None,
 ):
-    """Returns the expressibility :math:`\\|A\\|` of a parametrized
-    circuit, where
+    """Returns the expressibility :math:`\\|A\\|` of a parametrized circuit,
+    where.
```

Comment:
```suggestion
    """Returns the expressibility :math:`\\|A\\|` of a parametrized circuit,
    where
```
here it looks a bit funny